client/landing/faq.tsx

```diff
@@ -0,0 +1,457 @@
+  children,
+}: {
+  to: string
+  fragment: string
+  children: React.ReactNode
+}) {
+  return <a href={`${to}#${fragment}`}>{children}</a>
+}
+
+const Splash = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: ${background800};
+  overflow: auto;
+
+  & * {
+    user-select: text;
+  }
+`
+
+const LogoContainer = styled.div`
+  ${pageWidth};
+  position: absolute;
+  top: 38px;
+  left: calc(50% - 940px / 2);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: flex-start;
+  pointer-events: none;
+
+  @media screen and (max-width: 980px) {
+    max-width: 100%;
+    left: 0;
+    justify-content: center;
+    padding: 0 16px;
+  }
+`
+
+const Logo = styled.img`
+  margin-top: 8px;
+  pointer-events: none;
+
+  @media screen and (max-width: 980px) {
+    display: none;
+  }
+`
+
+const StyledLogoText = styled(LogoText)`
+  width: 100%;
+  max-width: 464px;
+  margin-top: 80px;
+  pointer-events: none;
+
+  @media screen and (max-width: 980px) {
+    margin-top: 24px;
+  }
+`
+
+const Intro = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  width: 100%;
+  margin-top: 142px;
+  background-color: ${background700};
+  box-shadow: ${shadowDef4dp};
+
+  @media screen and (max-width: 980px) {
+    margin-top: 86px;
+  }
+`
+
+const FaqHeaderContainer = styled.div`
+  ${pageWidth};
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+
+  @media screen and (max-width: 980px) {
+    justify-content: center;
+  }
+`
+
+const FaqHeader = styled.div`
+  ${headline1};
+  width: 100%;
+  max-width: 464px;
+  margin: 0;
+  padding: 64px 0;
+
+  @media screen and (max-width: 980px) {
+    padding: 24px 16px;
+  }
+`
+
+const FaqToc = styled.div`
+  ${pageWidth};
+  padding: 48px 0;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  font-size: 20px;
+  line-height: 1.5;
+  border-bottom: 1px solid ${colorDividers};
+
+  @media screen and (max-width: 980px) {
+    padding: 32px 16px;
+  }
+`
+
+const FaqTitle = styled.div`
+  ${headline4}
+`
+
+export function Faq() {
+  const { t } = useTranslation()
+  const lastHash = useRef<string>()
+
+  useLayoutEffect(() => {
+    const onHashChange = () => {
+      if (lastHash.current !== location.hash) {
+        if (location.hash !== '' || lastHash.current !== undefined) {
+          let id = location.hash.slice(1)
+          id = id === '' ? 'faqToc' : id
+          const element = document.getElementById(id)
+
+          if (element) {
+            element.scrollIntoView()
+          }
+        }
+        lastHash.current = location.hash
+      }
+    }
+    onHashChange()
+
+    window.addEventListener('hashchange', onHashChange)
+    return () => window.removeEventListener('hashchange', onHashChange)
+  }, [])
+
+  const questions = [
+    {
+      question: t(
+        'landing.faq.supportedBwVersionQuestion',
```

Comment:
Would prefer a name that doesn't include an acronym, i.e. `supportedStarcraftVersionQuestion`